modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SwiftCodegen.java

```diff
@@ -115,10 +117,10 @@ public SwiftCodegen() {
         typeMapping.put("string", "String");
         typeMapping.put("char", "Character");
         typeMapping.put("short", "Int");
```

Comment:
> So maybe keep the line but change to typeMapping.put("short", "Int16"); ?

We can but this won't work I suspect. Indeed we don't have the equivalent of 

https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java#L1165

for shorts... Honestly I think this support for short is not really functional anyway so that's why I preferred not to touch it.